### PR TITLE
Merge dev-bn into main(for Bengali localization live version)

### PR DESCRIPTION
**Pull request by mitul3737**
This PR is created to Merge `dev-bn` branch into `main`
Checklist to make localization live.

Tracking issue: #410
#879

 - [x] validate all files which are not in the localization directory (content/bn/*)
 - [x] check i18n/bn.toml exist 
 - [x] acknowledge to initial localization contributors list
 - [x] build project of [Bengali Localization Progress](https://github.com/cncf/glossary/projects/9)
 - [x] check Preview works
 - [x] check l10n configuration in config.toml

### CODE REVIEW

File: content/bn/_index.md
Comment (jihoon-seo) [inline content/bn/_index.md:20]: Please update this "Acknowledgements" section properly, according to the current English version,
while adding acknowledgements to initial Bengali localization contributors list properly too. 😊

[English version]
https://glossary.cncf.io/#acknowledgements
> The Cloud Native Glossary was initiated by the CNCF Marketing Committee (**Business Value Subcommittee**) and includes contributions from [Catherine Paganini](https://www.linkedin.com/in/catherinepaganini/en/), [Chris Aniszczyk](https://www.linkedin.com/in/caniszczyk/), [Daniel Jones](https://www.linkedin.com/in/danieljoneseb/?originalSubdomain=uk), [Jason Morgan](https://www.linkedin.com/in/jasonmorgan2/), [Katelin Ramer](https://www.linkedin.com/in/katelinramer/), [Mike Foster](https://www.linkedin.com/in/mfosterche/?originalSubdomain=ca), and many more contributors. For a complete contributor list, please refer to [this GitHub page](https://github.com/cncf/glossary/graphs/contributors).
> 
> The Glossary is maintained by [Catherine Paganini](https://www.linkedin.com/in/catherinepaganini/en/), [Jason Morgan](https://www.linkedin.com/in/jasonmorgan2/), [Jihoon Seo](https://www.linkedin.com/in/jihoon-seo/), [Noah Ispas](https://www.linkedin.com/in/noah-ispas-0665b42a/), and [Seokho Son](https://www.linkedin.com/in/seokho-son/).

**Review by jihoon-seo — COMMENTED** [review]
@mitul3737 PTAL!

**Comment by seokho-son** [comment]
To maintainers, please note that commits are not managed properly in dev-bn branch. 
There are duplicated commits. (the duplicated commits are from PR: [bn] Update dev-bn with main #879) 
So, we need to use the merge method `Squash and merge` in order to manage commits. (also considering there are meaningful commit messages)

**Review by Imtiaz1234 — COMMENTED** [review]
/lgtm

**Comment by seokho-son** [comment]
I am going to handle this PR after #844 merged. FIFO :)

File: content/bn/contribute/_index.md
Comment (jihoon-seo) [inline content/bn/contribute/_index.md:108]:
```suggestion
### একটি সমস্যার মাধ্যমে একটি পরিবর্তনের অনুরোধ করুন {#request-a-change-via-an-issue}
```

File: content/bn/contribute/_index.md
Comment (jihoon-seo) [inline content/bn/contribute/_index.md:117]:
```suggestion
### একটি টার্ম সরাসরি আপডেট করুন {#update-a-term-directly}
```

File: content/bn/contribute/_index.md
Comment (jihoon-seo) [inline content/bn/contribute/_index.md:124]:
```suggestion
## শব্দকোষ অনুবাদ করতে সাহায্য করুন {#help-translate-the-glossary}
```

File: content/bn/software_as_a_service.md
Comment (jihoon-seo) [inline content/bn/software_as_a_service.md:13]: Would it be better to use the Bengali word of "scale", rather than using the English word "scale" as-is?

For example, in [`cloud_computing.md` of this PR](https://github.com/cncf/glossary/pull/848/files#diff-4ce133fe6b2215efd14636f576fee90fcbea9ba9606ecb7a45fd59bc6b5587af),
I can see:
`[স্কেল(scale)](/scalability/)`

File: content/bn/devops.md
Comment (jihoon-seo) [inline content/bn/devops.md:11]: Would it be better to use the Bengali words of "tightly-coupled" and "monolithic apps", rather than using the English words "tightly-coupled" and "monolithic apps" as-is?

File: content/bn/contribute/_index.md
Comment (jihoon-seo) [inline content/bn/contribute/_index.md:61]: This is a list of some CNCF Glossary maintainers.

```suggestion
অনুগ্রহ করে রক্ষণাবেক্ষণকারীদের জানান যে আপনি স্ল্যাকে একটি মেয়াদ জমা দিয়েছেন কারণ তারা অন্যথায় এটি মিস করতে পারে। আদর্শভাবে, ট্যাগ করুন  _@Catherine Paganini_, _@jmo_, _@Seokho Son_, _@Jihoon Seo_ অথবা _@iamnoah_। যদি শব্দটি অনুমোদিত হয় এবং আপনি এটিতে কাজ করতে চান তবে তারা এটি আপনাকে বরাদ্দ করবে।
```

File: content/bn/contribute/_index.md
Comment (jihoon-seo) [inline content/bn/contribute/_index.md:37]: This is a list of some CNCF Glossary maintainers.

```suggestion
এছাড়াও, অনুগ্রহ করে CNCF Slack-এর [#glossary](https://cloud-native.slack.com/archives/C02TX20MQBB) চ্যানেলে যোগ দিন এবং রক্ষণাবেক্ষণকারীদের জানান যে আপনি একটি নতুন শব্দের জন্য একটি সমস্যা উত্থাপন করেছেন (আদর্শভাবে) , ট্যাগ করুন  _@Catherine Paganini_, _@jmo_, _@Seokho Son_, _@Jihoon Seo_, এবং/অথবা _@iamnoah_ যাতে তারা এটি মিস না করে)। মনে রাখবেন যে আপনি একবারে শুধুমাত্র একটি মেয়াদ দাবি করতে পারেন। আপনি যদি একাধিক শর্তে কাজ করতে চান, অনুগ্রহ করে পরেরটি দাবি করার আগে একটি শেষ করুন।
```

File: content/bn/cluster.md
Comment (jihoon-seo) [inline content/bn/cluster.md:9]: Let's fix a typo:

```suggestion
একটি ক্লাস্টার হল কম্পিউটার বা অ্যাপ্লিকেশনগুলির একটি গ্রুপ যা একটি সাধারণ লক্ষ্যে একসাথে কাজ করে। ক্লাউড নেটিভ কম্পিউটিং প্রসঙ্গে, শব্দটি প্রায়শই কুবারনেটে প্রয়োগ করা হয়। একটি Kubernetes ক্লাস্টার হল পরিষেবাগুলির একটি সেট (বা কাজের চাপ) যা তাদের নিজস্ব পাত্রে চলে, সাধারণত বিভিন্ন মেশিনে। এই সমস্ত [কন্টেইনারাইজড(Containerized)](/containerization/) পরিষেবাগুলির সংগ্রহ, একটি নেটওয়ার্কের মাধ্যমে সংযুক্ত, একটি ক্লাস্টার প্রতিনিধিত্ব করে।
```

**Review by jihoon-seo — COMMENTED** [review]
@mitul3737 PTAL!

**Comment by seokho-son** [comment]
The conflict has been resolved by #888

**Review by seokho-son — COMMENTED** [review]
Hi @mitul3737 

I am sorry, my review comment was pending. 
Please open another PR to resolve the issue related with Ack :)

File: content/bn/_index.md
Comment (mitul3737) [reply content/bn/_index.md:18]: Okay sure!

Comment (mitul3737) [reply content/bn/_index.md]: Updated the Acknowledgement @seokho-son

**Comment by seokho-son** [comment]
This PR seems ready to be merged !

@jihoon-seo please review and confirm this PR :)

Finally, I will check and confirm this PR, and let me merge. (will use squash and merge method)

**Review by jihoon-seo — APPROVED** [review]
LGTM!

**Review by seokho-son — APPROVED** [review]
Approve !! :)

**Pull request merged**
